test/e2e/seeder/network-states/withBalanceChecker.json

```diff
@@ -0,0 +1,12 @@
+      "storage": {
+        "": ""
```

Comment:
no need to change anything in the balance checker as this will call the ERC20 balance, so the balance for the tokens has to be modified only at the ERC20 contract